### ISSUE TITLE
Would it be possible to do bulk transfer by species?

### ISSUE THREAD

**Issue by sebastiangaertner** [Greenstand/treetracker-wallet-api#254]
Bulk transfer should be able to work with filters such as species, time frame, at some point shape files, planter_id etc. 
I assume this being a tricky task and it might actually rather be a selection tool that exports individual tokens based on these filters.

**Comment by dadiorchen** [comment]
Yes, interesting case, generally I agree with you, that we can use a tool to export the individuals and dump them into wallet API to do the job.

**Comment by dadiorchen** [comment]
Is there any really case/demand happen in the field? Like we need to do it this way with some org?

**Comment by sebastiangaertner** [comment]
@Kpoke its basically the filter function that we know is needed. When transferring tokens it is important to have a selection process. This ticket though is most likely a duplicate.
